### PR TITLE
Autoloop easy tag

### PR DESCRIPTION
**Pull request by GeorgeTsagk**
### Description

Based on https://github.com/lightninglabs/loop/pull/590

This PR adds a new label for easy autoloops. This helps us identify which loops were dispatched from easy-autoloop through the `user_agent` field

### CODE REVIEW

**Comment by hieblmi** [comment]
Could you briefly explain in the PR description what this new label is used for so new folks get an idea?

File: liquidity/liquidity.go [hunk 1]
```diff
@@ -1052,7 +1052,8 @@ func (m *Manager) checkExistingAutoLoops(ctx context.Context,
 	var summary existingAutoLoopSummary
 
 	for _, out := range loopOuts {
-		if out.Contract.Label != labels.AutoloopLabel(swap.TypeOut) {
+		if out.Contract.Label != labels.AutoloopLabel(swap.TypeOut) &&
+			out.Contract.Label != labels.EasyAutoloopLabel(swap.TypeOut) {
```
Comment (hieblmi) [inline liquidity/liquidity.go:1056]: nit: line length

maybe do something like 
```
label := out.Contract.Label
outLabel := labels.AutoloopLabel(swap.TypeOut)
easyOutLabel := labels.EasyAutoloopLabel(swap.TypeOut)
if label != outLabel && label != easyOutLabel {
	continue
}
```

File: liquidity/liquidity.go [hunk 2]
```diff
@@ -1082,7 +1083,8 @@ func (m *Manager) checkExistingAutoLoops(ctx context.Context,
 	}
 
 	for _, in := range loopIns {
-		if in.Contract.Label != labels.AutoloopLabel(swap.TypeIn) {
+		if in.Contract.Label != labels.AutoloopLabel(swap.TypeIn) &&
+			in.Contract.Label != labels.EasyAutoloopLabel(swap.TypeIn) {
```
Comment (hieblmi) [inline liquidity/liquidity.go:1087]: nit: line length, same as above

**Review by hieblmi — COMMENTED** [review]
Could you briefly explain in the PR description what this new label is used for so new folks get an idea?

File: liquidity/liquidity.go (hunk 1 above)
Comment (GeorgeTsagk) [reply liquidity/liquidity.go:1056]: I created a helper, as we might get even more labels in the future, so we can isolate all checks in there

**Review by hieblmi — APPROVED** [review]

**Pull request merged**